sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -219,4 +219,4 @@ def send(self, request: Union[HttpRequest, LegacyHttpRequest], **kwargs) -> Http
         except httpx.RequestError as err:
             raise ServiceRequestError(err, error=err) from err
 
-        return HttpXTransportResponse(request, response, stream_contextmanager=stream_ctx)
+        return HttpXTransportResponse(request, response, stream_contextmanager=stream_ctx) # pylint: disable=used-before-assignment
```

Comment:
can we move this to line 214, or do we not want an exception from it to be handled?